src/jailer/src/resource_limits.rs

```diff
@@ -187,4 +187,42 @@ mod tests {
         assert_eq!(rlim.rlim_cur, new_limit);
         assert_eq!(rlim.rlim_max, new_limit);
     }
+
+    #[test]
+    fn test_install() {
+        // Setup the resource limits
+        let mut rlimits = ResourceLimits::default();
+        let new_limit = 100;
+        rlimits.set_file_size(new_limit);
+        rlimits.set_no_file(new_limit);
+        
+        // Install the new limits to file size and 
+        // the number of file descriptors
+        let result = rlimits.install();
+        assert_eq!(result.unwrap(), ());
```

Comment:
nit: This assertion cannot fail, because the unit type `()` only allows a single value, `()`. Therefore, if we have a value of type unit, we know it must equal `()`. The important part here is the `.unwrap()`, which checks that we indeed did not get an error during execution:

```suggestion
        rlimits.install().unwrap();
```

Reply:
ah understood thank you Patrick. I assume that unwrap will also throw an error in failure cases?

Reply:
Yes, `.unwrap()` will `panic!` if the returned result contains an `Err` variant

Reply:
Thanks Patrick! pushed a change to fix this and fixed the formatting too.